pkgs/by-name/mi/minion/package.nix

```
@@ -0,0 +1,76 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+  openjfx21,
+  openjdk21,
+  makeDesktopItem,
+  wrapGAppsHook3,
+  makeBinaryWrapper,
+}:
+
+let
+  openjfx = openjfx21.override { withWebKit = true; };
+  openjdk = openjdk21.override {
+    enableJavaFX = true;
+    inherit openjfx;
```

The JDK derivation was changed recently, and the override doesn't eval anymore.
```suggestion
    openjfx_jdk = openjfx;
```